docs/README.md

```diff
@@ -24,101 +24,138 @@ kubectl -n gangway create secret generic gangway-key \
   --from-literal=sesssionkey=$(openssl rand -base64 32)
 ```
 
-## Docker image
+## Detailed Instructions
 
-A recent release of Gangway is available at
+The following guide is a more detailed review of how to get Gangway and other components configured.
```

Comment:
I am wondering if we should call out that these instructions assume infrastructure is on AWS.

Reply:
Yeah I guess Contour has the most aws specific references in the service annotation. I'll add a note to be clear.

Reply:
Sounds good.